Documentation/feature-band-source-building.md

```diff
@@ -0,0 +1,893 @@
+  from the 1xx band build that are redistributed across feature bands
+
+## Overview
+
+Building .NET SDK feature bands from source follows a two-stage approach:
+
+1. **Build the 1xx band** - This produces the shared .NET runtime and the 1xx
+   SDK tooling. The 1xx band contains all runtime components and serves as the
+   foundation for other bands.
+
+2. **Build additional bands (2xx, 3xx, etc.)** - These bands contain only the
+   SDK tooling components that differ from the 1xx band. They reuse the shared
+   runtime from the 1xx build.
+
+Each feature band branch in the VMR contains only the source code that differs
+from the 1xx band. Components that don't change between bands are excluded
+from the later band branches to avoid confusion and maintenance overhead.
+
+```mermaid
+flowchart LR
+    subgraph "1xx Band (Full VMR)"
+        Runtime[.NET Runtime]
+        Core[Core Libraries]
+        SDK1xx[1xx SDK Tools]
+    end
+    
+    subgraph "2xx Band (Subset VMR)"
+        SDK2xx[2xx SDK Tools]
+    end
+    
+    subgraph "3xx Band (Subset VMR)"
+        SDK3xx[3xx SDK Tools]
+    end
+    
+    subgraph "4xx Band (Subset VMR)"
+        SDK4xx[4xx SDK Tools]
+    end
+    
+    Runtime -.-> SDK2xx
+    Runtime -.-> SDK3xx
+    Runtime -.-> SDK4xx
+    Core -.-> SDK2xx
+    Core -.-> SDK3xx
+    Core -.-> SDK4xx
+    
+    SDK1xx --> SDK2xx
+    SDK2xx --> SDK3xx
+    SDK3xx --> SDK4xx
+```
+
+*Dotted lines indicate shared runtime components from the 1xx band; solid
+lines show SDK tooling dependencies between bands*
+
+## Understanding Bootstrap vs Sequential Build
+
+### Bootstrap Process (Two-Stage)
+
+Bootstrapping is required when you don't have suitable source-built artifacts
+to build a release. It's a two-stage process:
+
+1. **Stage 1**: Use Microsoft-provided source-built artifacts and Microsoft
+   SDK to build the target release
+2. **Stage 2**: Use the outputs from Stage 1 to rebuild the same release from
+   source
+
+The second stage validates that the release can be built using only
+source-built components and produces the final artifacts for distribution.
+
+```mermaid
+flowchart TD
+    subgraph Bootstrap["Bootstrap (Two-Stage)"]
+        MS_SDK[Microsoft SDK] --> Stage1[Stage 1 Build]
+        MS_Artifacts[Microsoft Source-Built<br/>Artifacts] --> Stage1
+        Stage1 --> SB_SDK[Source-Built SDK]
+        Stage1 --> SB_Artifacts[Source-Built Artifacts]
+        SB_SDK --> Stage2[Stage 2 Build]
+        SB_Artifacts --> Stage2
+        Stage2 --> Final1[Final Source-Built SDK]
+        Stage2 --> Final2[Final Source-Built<br/>Artifacts]
+    end
+```
+
+See [Bootstrapping Guidelines](bootstrapping-guidelines.md) for more info.
+
+### Sequential Build (Single-Stage)
+
+When you have appropriate source-built artifacts from previous builds, you can
+build directly without bootstrapping. This is the typical workflow for ongoing
+releases.
+
+```mermaid
```

Comment:
The title is truncated in my view 

<img width="1392" height="486" alt="Image" src="https://github.com/user-attachments/assets/be08b5b6-47ea-4e8f-a539-7f0af721945e" />